src/Json.Schema.ToDotNet/SyntaxNodeExtensions.cs

```diff
@@ -120,20 +120,17 @@ private static BaseTypeDeclarationSyntax AddGeneratedCodeAttribute(BaseTypeDecla
             AttributeListSyntax attributeListForGeneratedCodeAttribute =
                 SyntaxFactory.AttributeList(SyntaxFactory.SingletonSeparatedList(MakeGeneratedCodeAttribute()));
 
-            var enumDecl = typeDecl as EnumDeclarationSyntax;
-            if (enumDecl != null)
+            if (typeDecl is EnumDeclarationSyntax enumDecl)
```

Comment:
>enumDecl [](start = 50, length = 8)

Hygiene: Use C# 7 "pattern matching".

Reply:
TIL

---
In reply to: [244507593](https://github.com/Microsoft/jschema/pull/62#discussion_r244507593) [](ancestors = 244507593)

Reply:
Yeah, I learned that from one of the IDE diagnostics in the error window and read about it yesterday.

---
In reply to: [244508046](https://github.com/Microsoft/jschema/pull/62#discussion_r244508046) [](ancestors = 244508046,244507593)